src/Snake.DataAccess/UnitOfWork.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Snake.DataAccess.Repositories;
+using Snake.Game.Models;
+using System;
+
+namespace Snake.DataAccess
+{
+    public class UnitOfWork : IUnitOfWork, IDisposable
+    {
+        private SnakeDBContext _context;
+        private GenericRepository<User> userRepository;
+        private GenericRepository<Scores> scoresRepository;
+
+        public UnitOfWork(SnakeDBContext context)
+        {
+            _context = context;
+        }
+
+        public GenericRepository<User> UserRepository
+        {
+            get
+            {
+
+                if (this.userRepository == null)
+                {
+                    this.userRepository = new GenericRepository<User>(_context);
+                }
+                return userRepository;
+            }
+        }
+
+        public GenericRepository<Scores> ScoresRepository
+        {
+            get
+            {
+
+                if (this.scoresRepository == null)
+                {
+                    this.scoresRepository = new GenericRepository<Scores>(_context);
+                }
+                return scoresRepository;
+            }
+        }
+
+        public void Save()
+        {
+            _context.SaveChanges();
+        }
+
+        private bool disposed = false;
```

Comment:
Keep member ordering consistent. Private members should be placed before the constructor. See default code guidelines and conventions from Microsoft.